source/utils/student/test_my_work.py

```diff
@@ -183,7 +183,7 @@ def verifyWorkingDirectory(expectedPath: str) -> bool:
     if not verifyPythonVersion(MIN_VERSION, sys.version_info):  # type: ignore
         sys.exit(1)
         
-    if not verifyWorkingDirectory(os.path.abspath(__file__)):
+    if not verifyWorkingDirectory(os.path.dirname(os.path.abspath(__file__))):
```

Comment:
I should have caught this during code review -  we need to make sure that we are in the same directory as the file, which will always be false if we are checking the against the absolute path of the `test_my_work.py` script